src/os/windows/mod.rs

```diff
@@ -909,6 +909,17 @@ impl Window {
         }
     }
 
+    pub fn set_mouse_pos(&mut self, top: u32, left: u32) {
+        if left < self.width as u32 && top < self.height as u32 {
+            unsafe {
+                let pos = self.get_position();
+
+                winuser::SetCursorPos(left as i32 + pos.1 as i32, top as i32 + pos.0 as i32 + 30 /* height of title bar */);
```

Comment:
I don't think hard coding height to 30 here is a good idea. The window may not have a title bar, or the user may have increased / decreased the font size of the title bar.